packages/form/src/Checkbox.js

```diff
@@ -14,25 +15,27 @@ const Checkbox = ({
 }) => {
   const { value, toggle, metadata } = useCheckboxGroup(checkValue);
 
+  const inputId = id || uuid();
```

Comment:
Wouldn't this potentially generate a new uuid with every render?

Maybe there is (or we can make) a `useUuid()` which will generate the uuid on the first render and return the same value every subsequent render. (or `useId()` to be agnostic to the underlying lib if you go with nanoid)

Reply:
Actually, you are correct this most certainly will. Love the suggestion! We should just add this as a sub component under the `@availity/form` package as it could be useful for DX for people to use that so they can track the idea in a HOC for setting field values manually later.

Reply:
I suppose `const [id] = useState(() => uuid())` (or maybe even just `const [id] = useState(uuid)`, I think) would be similar. Doesn't seem like a specialized hook would be needed.

Reply:
Right, it's a simple implementation but a pretty useful hook nonetheless that can just be an addition to some hooks package rather than defining it ourselves.

being way to overkill here with my comments so you guys can just ignore me :upsidee

Reply:
I thought about adding it to react-use, but either have to pick the lib (uuid vs nanoid) or if we allow the developer to supply their own (to be more flexible), it ends up being a pointless abstraction (IMO) `const id = useId(uuid)` vs `const [id] = useState(uuid)`